connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/outbound/operation/ConnectorOperations.java

```diff
@@ -73,6 +73,11 @@ private static ParameterDescriptor map(Parameter parameter) {
           getVariableName(variableAnnotation), parameter.getType(), variableAnnotation.required());
     } else if (parameter.getType().equals(OutboundConnectorContext.class)) {
       return new Context();
+    } else if (parameter.isAnnotationPresent(Header.class)) {
+      io.camunda.connector.api.annotation.Header headerAnnotation =
+          parameter.getAnnotation(io.camunda.connector.api.annotation.Header.class);
```

Comment:
nit: fully qualified name is redundant as we import it above

Reply:
Thanks, fixed!